### PR TITLE
Dev 3164 digital signature drfo validation

### PR DESCRIPTION
**Pull request by progsmile**
edenlabllc/ehealth.api#3164

### CODE REVIEW

File: apps/core/config/dev.exs [hunk 1]
```diff
@@ -14,3 +14,5 @@ config :core, Core.Patients, pk_hash_salt: "aNg9JXF48uQrIjFYSGXDmKEYBXuu0BOEbkec
 
 config :core, :mongo, url: "mongodb://localhost:27017/medical_data"
 config :core, :mongo_audit_log, url: "mongodb://localhost:27017/medical_data"
+
+config :core, digital_signarure_enabled?: true
```
Comment (AlexKovalevych) [inline apps/core/config/dev.exs]: maybe its better to set it once in config.exs and override in prod.exs

Comment (progsmile) [reply apps/core/config/dev.exs]: ok

File: apps/api/test/api_web/controllers/encounter_controller_test.exs [hunk 2]
```diff
@@ -271,7 +270,9 @@ defmodule Api.Web.EncounterControllerTest do
 
   describe "cancel encounter" do
     setup %{conn: conn} do
-      expect_signature()
+      expect(IlMock, :get_dictionaries, 2, fn _, _ ->
```
Comment (AlexKovalevych) [inline apps/api/test/api_web/controllers/encounter_controller_test.exs]: this not needed anymore (rebase develop)

File: apps/core/test/unit/kafka/consumer/cancel_package_test.exs [hunk 3]
```diff
@@ -23,7 +24,9 @@ defmodule Core.Kafka.Consumer.CancelPackageTest do
 
   describe "consume cancel package event" do
     setup do
-      expect_signature()
+      expect(IlMock, :get_dictionaries, 2, fn _, _ ->
```
Comment (AlexKovalevych) [inline apps/core/test/unit/kafka/consumer/cancel_package_test.exs]: same here

File: apps/core/test/unit/kafka/consumer/cancel_package_test.exs [hunk 4]
```diff
@@ -276,7 +282,11 @@ defmodule Core.Kafka.Consumer.CancelPackageTest do
     test "diagnosis deactivated" do
       stub(KafkaMock, :publish_mongo_event, fn _event -> :ok end)
 
-      expect_signature()
+      expect(IlMock, :get_dictionaries, fn _, _ ->
```
Comment (AlexKovalevych) [inline apps/core/test/unit/kafka/consumer/cancel_package_test.exs]: same here

**Pull request merged**